packages/app-store/routing-forms/components/react-awesome-query-builder/widgets.tsx

```diff
@@ -118,7 +118,7 @@ const TextWidget = (props: TextLikeComponentPropsRAQB) => {
       containerClassName="w-full"
       type={type}
       value={textValue}
-      labelSrOnly={noLabel}
+      noLabel={noLabel}
```

Comment:
instead of hiding it from the normal user only(and still showing up for screen reader), hide the label completely.

This is because screen reader then sees duplicate labels. This was identified by getByRole selector in react-testing-library tests.